lib/tools/src/cover.erl

```diff
@@ -2525,11 +2523,64 @@ fill1() ->      "..|  ".
 fill2() ->       ".|  ".
 fill3() ->        "|  ".
 
+%% HTML sections
+create_header(OutFile, Enc) ->
+    ["<!doctype html>\n"
+    "<html>\n"
+    "<head>\n"
+    "<meta charset=\"",html_encoding(Enc),"\">\n"
+    "<title>",OutFile,"</title>\n"
+    "<style>"] ++
+    read_stylesheet() ++
+    ["</style>\n",
+    "</head>\n"
+    "<body>\n"
+    "<h1><code>",OutFile,"</code></h1>\n"].
+
+create_footer(ErlFile, Timestamp) ->
+    ["<footer>File generated from <code>",ErlFile,
+    "</code> by <a href=\"http://erlang.org/doc/man/cover.html\">cover</a> at ",
+    Timestamp,"</footer>\n<table>\n<tbody>\n"].
+
+close_html() ->
+    lists:concat(["</tbody>\n",
+                  "<thead>\n",
+                  "<tr>\n",
+                  "<th>Line</th>\n",
+                  "<th>Hits</th>\n",
+                  "<th>Source</th>\n",
+                  "</tr>\n",
+                  "</thead>\n",
+                  "</table>\n",
+                  "</body>\n"
+                  "</html>\n"]).
+
+table_row(CssClass, Line, L, N) ->
+    ["<tr class=\"",CssClass,"\">\n", table_data(Line, L, N)].
+table_row(Line, L) ->
+    ["<tr>\n", table_data(Line, L, "")].
+
+table_data(Line, L, N) ->
+   LineNoNL = Line -- "\n",
+   ["<td class=\"line\" id=\"L",integer_to_list(L),"\">",
+    integer_to_list(L),
+    "</td>\n",
+   "<td class=\"hits\">",maybe_integer_to_list(N),"</td>\n",
+   "<td class=\"source\"><code>",LineNoNL,"</code></td>\n</tr>\n"].
+
+maybe_integer_to_list(N) when is_integer(N) -> integer_to_list(N);
+maybe_integer_to_list(_) -> "".
+
+read_stylesheet() ->
+    PrivDir = code:priv_dir(?TOOLS_APP),
+    {ok, Css} = file:read_file(filename:join(PrivDir, ?STYLESHEET)),
+    lists:map(fun erlang:binary_to_list/1, binary:split(Css, <<"\n">>, [global])).
```

Comment:
I believe `cover` is working with iolists, so we don't need to break it apart on multiple lines. Returning `[Css]` should be fine.